Adyen/Analytics/AnalyticsProvider/AnalyticsProvider.swift

```diff
@@ -26,9 +26,8 @@ public struct AnalyticsConfiguration {
 
 @_spi(AdyenInternal)
 public protocol AnalyticsProviderProtocol: TelemetryTrackerProtocol {
-
-    /// :nodoc:
-    var checkoutAttemptId: String? { get }
+    
+    func fetchCheckoutAttemptId(completion: @escaping (String?) -> Void)
```

Comment:
no, the `AnalyticsPrvider` is marked with `@_spi(AdyenInternal)`, right? this is called by us internally

Reply:
Ah yeah, and the question was because I couldn't the implementation of `fetchCheckoutAttemptId` only to realize it existed and you just moved the signature into the protocol  :D